test/tests/dialog-modal_dialog.js

```diff
@@ -203,6 +204,13 @@ ariaTest('"aria-modal" attribute on role="dialog"', exampleFile, 'aria-modal', a
     await assertAttributeValues(t, ex.dialogSelector, 'aria-modal', 'true');
 });
 
+ariaTest('"aria-modal" attribute on nested role="dialog"', exampleFile, 'aria-modal', async (t) => {
+  t.plan(2);
```

Comment:
```suggestion
```

We've updated the tests slightly to no longer need `t.plan`, probably caught in your rebase :)